### PR TITLE
Added code to handle business partner migration no duplicates

### PR DESCRIPTION
**Pull request by anderslarsson**

### CODE REVIEW

File: src/client/components/Report/Report.jsx [hunk 1]
```diff
@@ -203,6 +203,15 @@ class Report extends Components.ContextComponent
         }
     }
 
+    adjustPartnerId_SHORTTERM = (partner) => {
+        const partnersToAdjust = ['BaneNorSF', 'SpordriftAS', 'WurthNorgeAS'];
+        const index = partnersToAdjust.indexOf(partner);
+        if (index !== -1) {
+            return partnersToAdjust[index] + '_cus';
```
Comment (MisterFossey) [inline src/client/components/Report/Report.jsx:210]: Ids need to be changed to the correct new customerIds (See comment in BPR-4352)

File: src/server/services/helpers.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+/**
+ * Adjust partner id for Business partner migration. Will be removed when full business partner is implemented.
+ * 
+ * @param {String} partnerId
+*/
+module.exports.adjustPartnerId_SHORTTERM = (partner) => {
+    const partnersToAdjust = ['BaneNorSF_cus', 'SpordriftAS_cus', 'WurthNorgeAS_cus'];
```
Comment (MisterFossey) [inline src/server/services/helpers.js:7]: Ids need to be changed to the correct new customerIds (See comment in BPR-4352)

File: test/server/reportService.spec.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+const config        = require('@opuscapita/config');
+const elasticsearch = require('elasticsearch');
+
+
+describe("reportService", () =>
+{
+    describe('adjustPartnerId_SHORTTERM', () => {
+        it('should return corrected partnerIds for Banenor, Spordrift and wurth not for others', () => {
+            const adjustPartnerId_SHORTTERM = require('../../src/server/services/helpers').adjustPartnerId_SHORTTERM;
+
```
Comment (MisterFossey) [inline test/server/reportService.spec.js:15]: Ids need to be changed to the correct new customerIds (See comment in BPR-4352)

File: test/server/reportService.spec.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+            const ReportService = require('../../src/server/services/ReportService');
+            this.reportService = new ReportService();
+            sinon.stub(this.reportService, 'createReportIndexIfNotExists').returns({});
+        }); 
+
+        it('Should call bnpService.getTransactions with old and new tenants and ReportServiceupsertElasticsearchDoc with old id', async () => {
+            sinon.stub(this.reportService, 'upsertElasticsearchDoc').returns();
+            
+            await this.reportService.init();
+            await this.reportService.createPartnerReport('2020-01-01', 'BaneNorSF_cus', 'invoiceReceiving');
```
Comment (MisterFossey) [inline test/server/reportService.spec.js:50]: Ids need to be changed to the correct new customerIds (See comment in BPR-4352)

File: src/client/components/Report/Report.jsx (hunk 1 above)
Comment (anderslarsson) [reply src/client/components/Report/Report.jsx:210]: Fixed

File: src/server/services/helpers.js (hunk 2 above)
Comment (anderslarsson) [reply src/server/services/helpers.js:7]: Fixed

File: test/server/reportService.spec.js (hunk 4 above)
Comment (anderslarsson) [reply test/server/reportService.spec.js:50]: fixed

**Pull request merged**